src/normalize.css

```
@@ -0,0 +1,17 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+.html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  line-height: 14px;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.body,
+p {
+  margin: 0;
+}
```

2 points:
- when resetting, it should be applied to tags, so that we cover all occurrences
- this part is rather resetting, so shouldn't be in a file with name _normalize.css_
```suggestion
body,
p {
  margin: 0;
}
```